src/resources/js/event-editor.js

```diff
@@ -3,21 +3,143 @@ var tribe_events_event_editor = tribe_events_event_editor || {};
 /**
  * Implements behaviours that are specific to the event editor.
  */
-jQuery( function( $ ) {
-	var obj = tribe_events_event_editor,
-		$sticky_in_month_view_checkbox = $( 'input[name="EventShowInCalendar"]' ),
-		$featured_event_checkbox = $( 'input[name="feature_event"]' );
+( function ( $, obj ) {
+	'use strict';
+
+	obj = obj || {};
+
+	/**
+	 *	Setup our selectors.
+	 *
+	 *	@since TBD
+	 */
+	obj.selectors = {
+		$sticky_in_month_view_checkbox: 'input[name="EventShowInCalendar"]',
+		$featured_event_checkbox: 'input[name="feature_event"]',
+		organizer: {
+			area: '#event_tribe_organizer',
+			delete_button: '.tribe-delete-this',
+			add_button: '#event_tribe_organizer .tribe-add-post',
+			post_dropdown: '.linked-post-dropdown',
+		},
+	};
+
+	obj.organizer = {};
+	/**
+	 *	Controls logic for the Organizer delete button to display.
+	 *
+	 *	If more than 1 organizer exists, display the delete button.
+	 *	If only 1 organizer exists, hide the delete button.
```

Comment:
```suggestion
	 *	If more than one organizer exists, display the delete button.
	 *	If only one organizer exists, hide the delete button.
```
Just better to not mix text and numbers - unless they _need_ to be numbers 😉